datalake-api/src/main/java/com/sequenceiq/sdx/api/endpoint/SdxEndpoint.java

```diff
@@ -165,4 +169,27 @@ SdxClusterResponse create(@ValidStackNameFormat @ValidStackNameLength @PathParam
     @ApiOperation(value = "list advertised datalake versions", produces = MediaType.APPLICATION_JSON, nickname = "advertisedruntimes")
     List<AdvertisedRuntime> advertisedRuntimes();
 
+    @POST
+    @Path("{name}/backupDatabase")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "backup the database backing datalake ", produces = MediaType.APPLICATION_JSON, nickname = "backupDatabase")
+    SdxDatabaseBackupResponse backupDatabaseByName(@PathParam("name") String name, @QueryParam("backupId") String backupId, @QueryParam("backupLocation") String backupLocation);
```

Comment:
the same here.. is it uuid or name?

Reply:
backupId is the UUID for the backup. It is the Id generated by datalake-dr service. Yes, the name is the stack name. There is no use for backup name here.